workflow/scripts/calc-precision-recall.py

```
@@ -185,22 +185,57 @@ def __init__(self, comparator, vaf_fields):
+    def _get_vaf_from_record(self, record, field, name):
+        """Extract VAF value from a record, handling FORMAT and INFO fields.
+
+        Returns a scalar float VAF (first value if array-like), or nan if not found.
+        """
+        import numpy as np
+        try:
+            if field == "INFO":
+                vaf = record.info.get(name)
+            else:
+                # FORMAT field - use samples dict
+                sample_name = list(record.samples.keys())[0]
```

Ruff (0.15.20)

[warning] Prefer `next(iter(record.samples.keys()))` over single element slice

Replace with `next(iter(record.samples.keys()))`

(RUF015)